melodic/distribution.yaml

```diff
@@ -2382,6 +2382,12 @@ repositories:
       url: https://github.com/ros/kdl_parser.git
       version: melodic-devel
     status: maintained
+  kobuki:
+    source:
+      type: git
+      url: https://github.com/yujinrobot/kobuki.git
+      version: melodic
```

Comment:
It's failing CI because there is no `melodic` branch on the repository to check out.